stonesoup/sensor/passive.py

```diff
@@ -33,13 +33,13 @@ class PassiveElevationBearing(Sensor):
             (and follow in format) the underlying \
             :class:`~.CartesianToElevationBearing` model")
 
-    def measure(self, ground_truth, noise=True, **kwargs):
+    def measure(self, ground_truths, noise=True, **kwargs):
         """Generate a measurement for a given state
 
         Parameters
         ----------
-        ground_truth : :class:`~.State`
-            A ground-truth state
+        ground_truths : :class:`~.State`
+            A set of :class:`~.GroundTruthState`
```

Comment:
These two seem inconsistent. Is it meant to be:

```suggestion
        ground_truths : :class:`~.GroundTruthState`
            A set of :class:`~.GroundTruthState`
```

Again, many places to update this...


Also, we need to update the return docs
``` 
        Returns
        -------
        Set[:class:`~.Detection`]
            A set of measurements generated from the given states. The timestamp of the
            measurement is set equal to that of the provided state.
```

Reply:
Actually, should this be:
```suggestion
        ground_truths : :class:Set [:class:`~.GroundTruthState`]
            A set of :class:`~.GroundTruthState`
```
or something similar (I haven't checked how that renders).